packages/charts/src/charts/line/utils/gradient.utils.ts

```diff
@@ -76,6 +78,8 @@ export const generateAreaGradient = (
     positiveColor: colorScale(positiveMax),
     zeroPointColor: colorScale(0),
     negativeColor: colorScale(positiveMin),
-    gradientZeroPercent: (positiveMax / (positiveMax + positiveMin)) * 100,
+    gradientZeroPercent: minAndMaxValueAreZero
+      ? 0
+      : (positiveMax / (positiveMax + positiveMin)) * 100,
```

Comment:
I think that we need to handle the case when `positiveMax + positiveMin = 0` as well

Reply:
I believe condition on `minAndMaxValueAreZero` handle this use case  ?

Reply:
There is no way to have `positiveMax + positiveMin = 0` because both are positive values.